mesos-docker/mesos-master/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM hubspot/mesos:1.0
```

Comment:
Can we plan to tag this after whatever the version of mesos we are building into it is? e.g. 1.9.0 or 1.10.0? Will keep it consistent with the other images and patterns we used to use